### DISCUSSION TITLE
How can I make sure a USB mount point is always the same?

### DISCUSSION THREAD

**Discussion by Myaanee** [Freetz-NG/freetz-ng#766] [Allgemein]
Currently freetz/avm generates the mount point based on USB Stick label, e.g `/­var/­media/­ftp/Fancy-Usb-Schtück-01`
however the tailing number is random sometimes thus breaking stuff that uses said path. Where can I change that behavior and make it always mount /dev/sda1 on a static path?

**Comment by fda77** [comment]
Freetz does not change this, that are avm scripts. If you remoe a device **unsave** the old mount point could not be reused. To remove save use the klickibuntui avm-webif

**Comment by AkteX** [comment]
Are you using any pathes to /­var/­media/­ftp/Fancy-Usb-Stick by some of the freetz packages? If yes it can occur that these pathes or files are created in the internal flash because the harddisk is not yet mounted. Then the harddisk can not be mounted to its mount point anymore. Is this the case? If yes you can use external processing and define the respective services as external services manually in the freetz webif. Such services will be started after the hard disk is already mounted.
